services/dynamic-sidecar/requirements/_base.txt

```diff
@@ -8,7 +8,7 @@ aiodebug==1.1.2
     # via
```

Comment:
Please avoid upgrading all these libraries in this PR.  In order to avoid unintended changes, just use the ``upgrade`` option in the make recipe:

- undo changes in ``_base.txt``, ``test.txt`` and ``tools.txt``
- add ``watchdog`` in ``_base.in`` (since this is done, just ``touch _base.in``)
- ``make reqs upgrade=watchdog`` should ONLY add ``watchdog`` (and all its dependencies) to the listings

Reply:
thanks, done that!